From 211e767ef8c1e27687b28974e35184b1632e3b65 Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Tue, 25 Feb 2020 14:24:57 +0800
Subject: [PATCH] cpufreq: add rockchip,rk3288w to blacklist

Some rk3288 chips have been writen special efuse, and uboot will
overwrite the compatible of dtb from rk3288 to rk3288w.

For cpufreq platdev driver, needs to add rk3288w to blacklist to
aviod generic cpufreq-dt driver work.

Change-Id: Ib079647b0ca483c9fbb729a7f81514a0d6a9931b
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 drivers/cpufreq/cpufreq-dt-platdev.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/cpufreq/cpufreq-dt-platdev.c b/drivers/cpufreq/cpufreq-dt-platdev.c
index ca8de2a26777..2392cd825185 100644
--- a/drivers/cpufreq/cpufreq-dt-platdev.c
+++ b/drivers/cpufreq/cpufreq-dt-platdev.c
@@ -118,6 +118,7 @@ static const struct of_device_id blacklist[] __initconst = {
 	{ .compatible = "rockchip,rk3228", },
 	{ .compatible = "rockchip,rk3229", },
 	{ .compatible = "rockchip,rk3288", },
+	{ .compatible = "rockchip,rk3288w", },
 	{ .compatible = "rockchip,rk3326", },
 	{ .compatible = "rockchip,rk3328", },
 	{ .compatible = "rockchip,rk3366", },
-- 
2.35.3

